src/go/api/scorch/app.go

```diff
@@ -159,7 +159,7 @@ func (this *Scorch) Running(ctx context.Context, exp *types.Experiment) error {
 		this.stopFilebeat(ctx, cmd, port)
 	}
 
-	if err := this.recordInfo(runID, runDir, exp.Metadata.Name, start); err != nil {
+	if err := this.recordInfo(runID, runDir, exp.Metadata.Name, exp.Metadata.Annotations["phenix.workflow/tags"], start); err != nil {
```

Comment:
Since we're now passing more than one experiment metadata related variable, it might make sense to just pass a single experiment metadata variable to the `recordInfo` function and let it pull the relevant keys out of the metadata.